### DISCUSSION TITLE
Any way to get a single assembled board at a reasonable price?

### DISCUSSION THREAD

**Discussion by obones** [vroland/epdiy#305] [Q&A]
Hello,

Many thanks for this project, it is really awesome to be able to use some relatively cheap displays for custom projects.
I uploaded the hardware files to JCLPCB and tried to get the minimum possible price and I came up with about 100€ for 5 boards, 2 assembled (without the ESP32) which is quite nice when brought back to a single board (~20€)
But my issue here is that I only really need a single board and I would not want the extra ones to go to waste.
Would there be a "marketplace" of people interested in PCB sharing?

Thanks for any suggestion

**Comment by vroland** [comment]
@martinberlin has some on tindie every once in a while :)

**Comment by martinberlin** [comment]
Thanks @vroland actually some boards are already submitted at 50% discount price there in the [flea](https://www.tindie.com/products/fasani/kindle-paperwhite-1-parallel-eink-s3-controller/) market.
Actually the boards I sell in tindie are at least 60€, except those I mention of course, since we solder some components manually and there is a double test before any board is sent to a client.
I usually offer this same finished board at half the price to epdiy collaborators too.
But I would like people asking for boards, I was one of them a year ago, that is not only sending them to fabrication but also dealing with issues post-fabrication like I mentioned in #304 . 
Of course I could sell a fully tested board for 35€ and be 0 gain. I also need to have a winning for the dedication, time spent, testing quality and so on. This is also time that I don't dedicate to my family and that's why boards finished and tested are that price.
That's why I encourage everyone asking for boards to understand the basics of KiCad and also the logistics of getting some of the components in "My Parts" because at some point many will be out of stock. And I don't really want to be the only one that has them in his library, but all the developers that want to make something with epdiy, should get at least some TPS65185 in their parts. So there is enough components available if you have a client request.
But I will always reserve cheaper boards for collaborators and also put money aside to collaborate

**Reply by obones** [reply]
Thanks for the explanations, I did not think so many boards would come with an issue from the manufacturers.
I'll keep your offers in mind, I must now decide which screen size is right, according to my family members, the EPD4.7 that I have here is a bit small.

**Reply by martinberlin** [reply]
Mostly is my own soldering errors since MCU I solder myself. But exceptionally it might be a fabrication error and once every 30 assembly rounds they might put a chip 180 degrees rotated. We all make little mistakes sometimes and fabrication it’s no exception.

But overall JLCPCB is really good and very good price if you use economic placement. I highly recommend them. But if you want the esp32S3 soldered then you will have to use Standard PCBA and then prices are much higher (so it’s also the soldering quality and MCU will look professionally soldered)
Sadly economic assembly is out for most esp32* modules so you either solder them by hand or you have to jump to standard assembly.
I highly recommend a 9.7 display or if you want to go bigger but also more expensive a 13.3 inches ES133UT2. Also if you want to have a 6” that is much nicer size compared to 4.7 you can get one of my discount boards at 35€ that are tested and fully working (just has 34 pins connector for 6 inches 1024*758)
